Control.Geocoder.css

```diff
@@ -48,7 +48,7 @@
   height: 26px;
   border: none;
   background-color: white;
-  background-image: url(images/geocoder.png);
+  background-image: url(images/geocoder.svg);
```

Comment:
We could (should?) even embed the SVG as data URL here, see https://css-tricks.com/lodge/svg/09-svg-data-uris/. What do you think?

Reply:
Yes, we can try this. I will change my PR.

Reply:
I changed it to data URLs.